tests/containers/podman_pods.pm

```diff
@@ -18,6 +18,43 @@ use serial_terminal 'select_serial_terminal';
 use version_utils qw(is_sle is_opensuse is_sle_micro);
 use containers::k8s qw(install_k3s uninstall_k3s);
 
+sub check_container_nspid {
+    # expect set hostpid
+    my $hostpid = shift;
+    my $config = 'hello-kubic.yaml';
+    my @errors = ();
+
+    record_info('NSPid', sprintf('Expect HostPid == %s', !!$hostpid ? 'True' : 'False'));
+
+    if ($hostpid) {
+        assert_script_run(qq[sed -i '/containers:/i\\      hostPID: true' $config]);
+    }
+
+    assert_script_run("podman play kube --replace $config");
+    my $pod = script_output(q[podman pod ps --format '{{ .Name }}']);
+    my $container = script_output(q[podman container ps --filter name=.*pod.* --format '{{ .Names }}']);
+    my $pid_on_host = script_output(qq[podman container inspect $container --format '{{ .State.Pid }}']);
```

Comment:
I'd additionally suggest we can also try `podman exec <container_name> echo $$`, which, again, should never be 1 as long as `hostPID` is set to true.

Reply:
[The image](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/data/containers/hello-kubic.yaml#L17) that we are using for this test does not come with `echo` command, unfortunately.

Reply:
Weird, I'm able to run this `podman run registry.opensuse.org/kubic/hello-kubic:latest echo $$` with expected results. Can you share what error you're facing?